### ISSUE TITLE
Many functions have non-documented arguments

### ISSUE THREAD

**Issue by Kdreval** [morinlab/GAMBLR#173]
```
❯ checking Rd \usage sections ... WARNING
  Undocumented arguments in documentation object 'annotate_recurrent_cnv'
    ‘seg_df’ ‘seg_file’
  Documented arguments not in \usage in documentation object 'annotate_recurrent_cnv':
    ‘seq_df’ ‘seq_file’
  
  Undocumented arguments in documentation object 'collate_csr_results'
    ‘seq_type_filter’
  
  Documented arguments not in \usage in documentation object 'fancy_qc_plot':
    ‘sort_by’
  
  Undocumented arguments in documentation object 'fancy_v_sizedis'
    ‘plot_trim’
  Documented arguments not in \usage in documentation object 'fancy_v_sizedis':
    ‘trim’
  
  Undocumented arguments in documentation object 'fetch_output_files'
    ‘unix_group’ ‘build’
  Documented arguments not in \usage in documentation object 'fetch_output_files':
    ‘genome_build’
  
  Undocumented arguments in documentation object 'finalize_study'
    ‘seq_type_filter’
  
  Undocumented arguments in documentation object 'get_gene_cn_and_expression'
    ‘gene_symbol’ ‘ensembl_id’
  Documented arguments not in \usage in documentation object 'get_gene_cn_and_expression':
    ‘hugo_symbol’ ‘ensembl_gene_id’
  
  Undocumented arguments in documentation object 'get_manta_sv'
    ‘pairing_status’
  Documented arguments not in \usage in documentation object 'get_manta_sv':
    ‘pair_status’
  
  Undocumented arguments in documentation object 'liftover_bedpe'
    ‘col_names’ ‘col_types’ ‘standard_bed’
  
  Undocumented arguments in documentation object 'populate_each_tool_result'
    ‘genome_builds’ ‘unix_groups’
  Documented arguments not in \usage in documentation object 'populate_each_tool_result':
    ‘genome_build’ ‘unix_group’
  
  Documented arguments not in \usage in documentation object 'populate_tool_results':
    ‘tool’
  
  Undocumented arguments in documentation object 'prettyCoOncoplot'
    ‘...’
  
  Undocumented arguments in documentation object 'prettyOncoplot'
    ‘arrange_descending’ ‘tally_all_mutations’ ‘tally_all_mutations_max’
  
  Undocumented arguments in documentation object 'process_all_manta_bedpe'
    ‘group’
  
  Undocumented arguments in documentation object 'setup_study'
    ‘seq_type_filter’
  
  Functions with \usage entries need to have the appropriate \alias
  entries, and all their arguments documented.
  The \usage entries must correspond to syntactically valid R code.
  See chapter ‘Writing R documentation files’ in the ‘Writing R
  Extensions’ manual.
```

**Comment by Kdreval** [comment]
This is also related here

```
❯ checking Rd contents ... WARNING
  Argument items with no description in Rd object 'trim_scale_expression':
    ‘x’
```

**Comment by mattssca** [comment]
Oh, I wish I knew that this check would return this. Would've been very helpful when going over the doc. I'll self-assign this and take care of this. Thanks, Kostia!

**Comment by Kdreval** [comment]
Thanks Adam!
This also relates:

```N  checking Rd files (940ms)
   prepare_Rd: check_gambl_metadata.Rd:23-25: Dropping empty section \value
   prepare_Rd: check_gamblr_config.Rd:23-25: Dropping empty section \value
   prepare_Rd: count_ssm_by_region.Rd:35-37: Dropping empty section \value
   prepare_Rd: find_expected_outputs.Rd:29-31: Dropping empty section \value
   prepare_Rd: get_samples_md5_hash.Rd:38-40: Dropping empty section \examples
   prepare_Rd: plot_multi_timepoint.Rd:26-28: Dropping empty section \value
   prepare_Rd: plot_mutation_dynamics_heatmap.Rd:66-68: Dropping empty section \examples
   prepare_Rd: process_all_manta_bedpe.Rd:24-26: Dropping empty section \value
   prepare_Rd: process_all_manta_bedpe.Rd:33-35: Dropping empty section \examples
   prepare_Rd: read_merge_manta_with_liftover.Rd:20-22: Dropping empty section \value
   prepare_Rd: tidy_gene_expression.Rd:12-14: Dropping empty section \value
   prepare_Rd: web_initialize_gambl_site.Rd:23-25: Dropping empty section \value
   prepare_Rd: write_sample_set_hash.Rd:14-16: Dropping empty section \value
   prepare_Rd: write_sample_set_hash.Rd:25-27: Dropping empty section \examples
```

We need to populate these or drop them from documentation

**Comment by Kdreval** [comment]
Resolved in #176
